Cargo.toml

```diff
@@ -130,3 +119,19 @@ optional = true
 branch = "pgbouncer-mode"
 git = "https://github.com/pimeys/rust-postgres"
 optional = true
+
+[dependencies.libsqlite3-sys]
+version = "0.18"
+default-features = false
+features = ["bundled"]
+optional = true
+
+[dependencies.tokio]
+version = "0.2"
+features = ["rt-threaded", "macros", "sync"]
+optional = true
+
+[dependencies.tokio-util]
+version = "0.3"
+features = ["compat"]
```

Comment:
Is this still needed?

Reply:
For tinerius

Reply:
tiberius runs on tokio 0.2?

Reply:
It's `tokio-util`, not `tokio`. Latest `tokio-util` is 0.4!